waterlog/src/reducers/index.js

```diff
@@ -11,7 +11,9 @@ import WastageGraphSeasonallyReducer from './WastageGraphSeasonallyReducer';
 import PumpsReducer from './PumpsReducer';
 import TankLevelReducer from './TankLevelReducer';
 import TankLevelsByIdReducer from './TankLevelsByIdReducer';
-
+import DailyCostReducer from './DailyCostReducer';
```

Comment:
Like I've mentioned earlier, can't you make one reducer that handles Daily, Monthly and Seasonal dispatches? It will save some space. They all look a lot alike.

Reply:
Thanks, I did not do them in  the same reducer because I still have to update each reducer with new actions.